### PR TITLE
add ma style buff display widget

### PR DESCRIPTION
**Pull request by dikiy4eburator**
#### Summary
Content "widget that shows the buffs and dbuffs from the martial art style"

#### Purpose of change
increase quality of life

#### Describe the solution
it works like the  "bp_status_legend_text" widget, but it displays the buffs from martial arts, that are usualy hidden in the character menu (@) but actualy esential for melle fights.

ma_show
(right of the pic)

#### Testing
testet it in an actual build

### CODE REVIEW

**Comment by bombasticSlacks** [comment]
You need to have a summary line like:

Content "widget that shows the buffs and dbuffs from the martial art style"

and everything else you currently have in summary needs to be moved to other sections

**Comment by bombasticSlacks** [comment]
It looks like this isn't actually included in any of the sidebars by default?

**Comment by dikiy4eburator** [comment]
> It looks like this isn't actually included in any of the sidebars by default?

yes, i only wanted it to be added and use it later.
im working at a custom bar now, where i want it to be used.
and becaus this is my first commit. i dont wantet to go to deep.

File: data/json/ui/ma_style.json [hunk 1]
```diff
@@ -11,5 +11,573 @@
...
+        "id": "buff_venom_snake_oncrit",
+        "text": "Viper's Venom"
+      },
+      {
+        "condition": { "u_has_effect": "mabuff:buff_venom_snake_onhit" },
+        "id": "buff_venom_snake_onhit",
+        "text": "Viper's Rattle"
+      }
+    ],
+    "default_clause": { "text": "no martial art activ" }
```
Comment (Rivet-the-Zombie) [inline data/json/ui/ma_style.json:570]: This case of 'activ' shoild be 'active' instead.

**Comment by dikiy4eburator** [comment]
may I ask a question?

WTF is this? the build broke because of three dots in a string in a JSON file?

Json error: data/json/ui/ma_style.json:570:36: ellipsis preferred over three dots.
    Suggested fix: replace "..." with "…"
    At the following position (marked with caret)

**Comment by anothersimulacrum** [comment]
We have a textual standard to use ellipses instead of three consecutive dots. This is enforced through the tests. It erroneously believed that your 3 dots were meant to be that.

**Comment by dikiy4eburator** [comment]
> We have a textual standard to use ellipses instead of three consecutive dots. This is enforced through the tests. It erroneously believed that your 3 dots were meant to be that.

Thx, never thought that there is something like an "ellipsis".
But is this standart only cosmetic nature, or does it matter for the code.

**Comment by anothersimulacrum** [comment]
Any string that is displayed to the player should use the ellipsis character instead of three dots. It does not matter for any string that is not displayed to the character.

**Comment by dikiy4eburator** [comment]
two short question.
is it actualy ok when some testf fail, can the brunch stil be merged?
or should i "push" untill all tests are passed?

**Comment by anothersimulacrum** [comment]
Tests can fail unless they are in the required runs or are failing because of your changes.

**Comment by vetall812** [comment]
Hi, could you give me some docs about this. Or probably point me to the examples in current sidebars. 
I'd like to add this to mine )

**Pull request merged**